main.go

```diff
@@ -69,7 +74,8 @@ func main() {
 		return nil
 	}
 
-	app.Action = runAction
+	// Default to 'start' is no command is specified
+	app.Action = startCommand.Action
```

Comment:
I didn't want to break existing functionality - meaning "runc" by itself. I believe this line makes it default to "start" if no command is given. Is that not correct?